packages/@react-native-windows/codegen/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "react-native-windows-codegen",
-  "version": "1.1.14",
+  "name": "@react-native-windows/codegen",
+  "version": "0.0.0-canary.0",
```

Comment:
Just checking as my understanding is that this will make the package no longer an 'evergreen' pacakge where the latest will always be used by customers...

Reply:
It will make it:

canary: evergreen
stable: versioned roughly along RNW releases